canard.h

```diff
@@ -356,6 +356,12 @@ void canardSetLocalNodeID(CanardInstance* ins,
  */
 uint8_t canardGetLocalNodeID(const CanardInstance* ins);
 
+/**
+ * Forgets the current node ID value so that a new Node ID can be assigned.
+ * This is useful for Battery Packs that are never power cycled, but need to be added and removed from networks regularly.
+ */
+void canardForgetLocalNodeID(CanardInstance* ins);
```

Comment:
Why does `forget` have to be a keyword here? To me this implies setting it to NULL. Should this not simply be something more general such as:

```
 * Resets the current node ID value to initial broadcast id, so that a new Node ID can be assigned.
 * Example use case: battery Packs that are never power cycled, but need to be added and removed from networks regularly.

canardResetLocallNodeID(CanardInstance* ins) 
```

Reply:
I also kind of like "reset" over "forget" because it does, in fact, reset to the default `node_id` value set in `canardInit()`, but I personally think the latter is acceptable since the function definition is so clear and a node can't _not_ have a node ID.

Reply:
Perhaps.  We can certainly do that (and I'm not opposed to it).  This isn't my commit per se.  This was implemented by someone who is no longer here.  So I cannot answer "why?"

Reply:
@viet-nguyen-matternet @afbowen Do you guys want this name change (canardResetLocalNodeID) or is this okay?

Reply:
I would prefer if it said: `canResetLocalNodeID`. If it isn't too much to change it.